master/reference/cni-plugin/configuration.md

```diff
@@ -71,11 +71,18 @@ The following deprecated options are also supported
 
 When using Calico IPAM, the following flags determine what IP addresses should be assigned. NOTE: These flags are strings and not boolean values.
 
-* `assign_ipv4` (default `"true"`)
-* `assign_ipv6` (default `"false"`)
+* `assign_ipv4` (default: `"true"`)
+* `assign_ipv6` (default: `"false"`)
 
 A specific IP address can be chosen by using [`CNI_ARGS`](https://github.com/appc/cni/blob/master/SPEC.md#parameters) and setting `IP` to the desired value.
 
+The list of possible IPv4 and IPv6 pools can also be specified via the following properties:
+
+* `ipv4_pools`: An array of CIDR strings (e.g. `"ipv4_pools": ["10.0.0.0/24", "20.0.0.0/16"]`)
+* `ipv6_pools`: An array of CIDR strings (e.g. `"ipv6_pools": ["2001:db8::1/120"]`) - Note: `assign_ipv6` needs to be set to `true` for IPv6 ipPools to work.
+
+Specified IP Pool needs to exist before this configuration can be applied. An error will be generated if one or more IP Pools specified do not already exist. 
```

Comment:
Wording is a bit off.  Should be "_The_ specified IP _Pools_ _need_".

How about this: "Any IP Pools specified in the CNI config must have already been created.  It is an error to specify IP Pools in the config that do not exist.  See [creating IP Pools][(link)."